molo/commenting/tests/test_models.py

```diff
@@ -32,3 +41,32 @@ def test_parent(self):
         second_comment.save()
         [child] = first_comment.children.all()
         self.assertEqual(child, second_comment)
+
+    def test_auto_remove_off(self):
+        comment = self.mk_comment('first comment')
+        comment.save()
+        comment_flag = self.mk_comment_flag(comment)
+        comment_flag.save()
+        signals.comment_was_flagged.send(
+            sender=comment.__class__,
+            comment=comment,
+            flag=CommentFlag.MODERATOR_DELETION,
+            created=True,
+        )
+        altered_comment = MoloComment.objects.get(pk=comment.pk)
+        self.assertFalse(altered_comment.is_removed)
+
+    def test_auto_remove_on(self):
+        comment = self.mk_comment('first comment')
+        comment.save()
+        comment_flag = self.mk_comment_flag(comment)
+        comment_flag.save()
+        settings.COMMENTS_FLAG_THRESHHOLD = 1
```

Comment:
@KaitCrawford The recommended way to do this is to use [override settings](https://docs.djangoproject.com/en/1.8/topics/testing/tools/#overriding-settings)